arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1521,6 +1521,17 @@
           "type": "boolean",
           "description": "EnableBgp flag"
         },
+        "UsePolicyBasedTrafficSelectors": {
```

Comment:
I would like to callout few things:
1. These changes are applied to an api-version for which I believe an SDK has already been published. This will call breaking changes in the sdk
2. The new properties that you have added are in PascalCase. All the other properties are in camelCase. Please double check what is present on the wire using Fiddler. Looking at the server side code in C# will not help.

@deepakrajendranmsft - Please take a look.